background/services/indexing/db.ts

```diff
@@ -287,22 +299,24 @@ export class IndexingDatabase extends Dexie {
     )
   }
 
-  async getCustomAssetsByNetworks(
+  async getActiveCustomAssetsByNetworks(
     networks: EVMNetwork[]
   ): Promise<SmartContractFungibleAsset[]> {
```

Comment:
```suggestion
  ): Promise<CustomAsset[]> {
```
shouldn't it return custom assets?